### DISCUSSION TITLE
Update native-models

### DISCUSSION THREAD

**Discussion by PhilippeThouvenot** [lunarstorm/laravel-ddd#67] [General]
Hi,
Could you update native-models branch with last version ?
Thank you.

**Comment by jaspertey** [comment]
That branch contains a bunch of experimental work in progress. I will be resuming work on it again later this year, but as of now, it should not be used or relied on. What were you doing with that branch?

**Comment by PhilippeThouvenot** [comment]
I use it because I find it more enjoyable to use. It works for what I do with it. I hope it will be finished one day.
For now I use a post-script in composer.json to fix the code.

**Reply by jaspertey** [reply]
I merged the latest changes from main into the branch earlier.

Glad to hear you’re finding that branch useful… I do plan on finishing it, maybe a little sooner now knowing there’s a demand for it!

**Comment by jaspertey** [comment]
native-models branch has been deleted, in favour of the official 1.2 release: https://github.com/lunarstorm/laravel-ddd/releases/tag/v1.2.0
